tools/defaults2.bzl

```diff
@@ -73,3 +75,32 @@ def jasmine_test(data = [], args = [], **kwargs):
         ],
         **kwargs
     )
+
+def karma_web_test_suite(name, **kwargs):
+    test_deps = kwargs.get("deps", [])
+
+    kwargs["tags"] = ["partial-compilation-integration"] + kwargs.get("tags", [])
```

Comment:
I think this is preference thing, but I prefer not to overwrite what is in the kwargs, but instead, for things from kwargs that get used, you `pop` them into local variables and provide those as params in the future usage.

But whatever you prefer can work here, there might be something else I don't know/understand at play as well.

Reply:
I just copied this over. I will update it as I also typically don't use this pattern anymore.